### ISSUE TITLE
Django 2.0 Warning: use_for_related_fields is deprecated

### ISSUE THREAD

**Issue by clokep** [jazzband/django-model-utils#290]
## Problem

Using the `InheritanceManager` on a class causes the following warning:

```
/usr/local/venv/local/lib/python2.7/site-packages/django/db/models/fields/related_descriptors.py:157: RemovedInDjango20Warning: use_for_related_fields is deprecated, instead set Meta.base_manager_name on 'my_app.MyModel'.
  qs = self.get_queryset(instance=instance)
```

This seems to be used in a couple of places:
* [`InheritanceManagerMixin`](https://github.com/jazzband/django-model-utils/blob/716e29d83bad90b59373383f9782d05ba089ea8c/model_utils/managers.py#L238)
* [`QueryManagerMixin`](https://github.com/jazzband/django-model-utils/blob/716e29d83bad90b59373383f9782d05ba089ea8c/model_utils/managers.py#L256)

I'm not really sure of the ramifications of just removing this vs. using `Meta.base_manager_name`, etc. I know there's some special things done with the first `Manager` found on a Django `Model` class.

## Environment

- Django Model Utils version: 3.0.0
- Django version:  1.11.5
- Python version: 2.7.6

**Comment by tony** [comment]
We need a game plan for this then. :smile: 2.0 was released yesterday

**Comment by tony** [comment]
Possibly related:

- https://github.com/deschler/django-modeltranslation/issues/389
- https://github.com/deschler/django-modeltranslation/pull/431

**Comment by tony** [comment]
More lore:

https://github.com/django/django/commit/ed0ff913c648b16c4471fc9a9441d1ee48cb5420

Old API:

```python
class CustomManager(models.Model):
    use_for_related_fields = True

class Model(models.Model):
    custom_manager = CustomManager()
```

New API:

```python
class Model(models.Model):
    custom_manager = CustomManager()

    class Meta:
        base_manager_name = 'custom_manager'
````

**Comment by bahendri** [comment]
Seems like this issue should be closed in light of `c4d7212`. 

(In my app, `pytest` threw this deprecation warning and a quick read seemed to suggest the problem was in this library but I later realized it was my app's fault. If this ticket had been marked closed I might have realized my mistake sooner.)
